modules/scheduler/gke-cluster/variables.tf

```diff
@@ -362,18 +368,20 @@ variable "cluster_reference_type" {
   description = "How the google_container_node_pool.system_node_pools refers to the cluster. Possible values are: {SELF_LINK, NAME}"
   default     = "SELF_LINK"
   type        = string
+  validation {
```

Comment:
Add `nullable = false` here otherwise the `contains` test can Error (stacktrace):

```
❯ terraform console
>  contains(["SELF_LINK", "NAME"], null)
╷
│ Error: Invalid function argument
│
│   on <console-input> line 1:
│   (source code not available)
│
│ Invalid value for "value" parameter: argument must not be null.
```

Reply:
Done